Integrations/integration-Cloaken.yml

```diff
@@ -0,0 +1,79 @@
+description: 'Unshorten urls through a tor proxy server. Unshorten urls onsite with
+  the power of Tor to prevent leaking ip addresses to adversaries.  Created by CypherInt.  Visit
+  us at https://cypherint.com.  Subscribe on the Amazon marketplace: https://aws.amazon.com/marketplace/'
+detaileddescription: cloaken will return the unshortened version of a url
+configuration:
+- display: Username
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Server URL
+  name: server_url
+  defaultvalue: ""
+  type: 0
+  required: true
+script:
+  script: |-
+    # The command demisto.command() holds the command sent from the user.
+    from cloakensdk.client import SyncClient
+    from cloakensdk.resources import Url
+
+    def get_client():
+        server=demisto.params()["server_url"]
+        password=demisto.params()["credentials"]["password"]
+        username=demisto.params()["credentials"]["identifier"]
+        client=SyncClient(server_url=server,
+                          username=username,
+                          password=password)
+        return  client
+
+
+    if demisto.command() == 'test-module':
+        # This is the call made when pressing the integration test button.
+        client = get_client()
+        demisto.results('ok')
+
+    if demisto.command() == "cloaken":
+        client = get_client()
+        url = demisto.args()["url"]
+        resource = Url(client)
+        resource.unshorten(url)
+        response = resource.full_request()
+
+        if response.get("status","FAILED")=="Success":
+            url_data=response["data"]["unshortened_url"]
+            ec={"URL":{"Data":url_data},"Cloaken":[]}
+            context={"original_url":url,"unshortened_url":url_data}
+            ec["Cloaken"].append(context)
```

Comment:
No need to initialise and then fill, do it one operation please

Reply:
done